pootle/apps/pootle_app/management/commands/calculate_checks.py

```diff
@@ -27,28 +27,30 @@ class Command(PootleCommand):
     help = "Allow checks to be recalculated manually."
 
     shared_option_list = (
-        make_option('--check', action='append', dest='check_names',
-                    help='Check to recalculate'),
+        make_option(
+            '--check',
+            action='append',
+            dest='check_names',
+            default=None,
+            help='Check to recalculate'
+        ),
     )
 
     cached_methods = [CachedMethods.CHECKS]
     process_disabled_projects = True
 
     def handle_all_stores(self, translation_project, **options):
-        check_names = options.get('check_names', None)
-        calculate_checks(check_names=check_names,
+        calculate_checks(check_names=options['check_names'],
                          translation_project=translation_project)
 
     def handle_all(self, **options):
         if not self.projects and not self.languages:
             logging.info(u"Running %s (noargs)", self.name)
 
-            check_names = options.get('check_names', None)
-            translation_project = options.get('translation_project', None)
             try:
 
-                calculate_checks(check_names=check_names,
-                                 translation_project=translation_project)
+                calculate_checks(check_names=options['check_names'],
+                                 translation_project=options['translation_project'])
```

Comment:
Where do this `translation_project` comes from? I couldn't this on `PootleCommand`.